rules-java/tests/src/test/java/org/jboss/windup/rules/apps/java/DiscoverAdditionalProjectDetailsTest.java

```diff
@@ -93,6 +94,9 @@ public void testLog4jNameFound() throws Exception
             boolean jarVersionFound = false;
             for (ArchiveModel archiveModel : archiveService.findAllByProperty(ArchiveModel.ARCHIVE_NAME, "log4j-1.2.6.jar"))
             {
+                if (archiveModel instanceof DuplicateArchiveModel)
+                    archiveModel = ((DuplicateArchiveModel)archiveModel).getOriginalArchive();
+
```

Comment:
We still identify project information for 3rd party libs. Without this change the code tries to do some illogical things as this test has multiple log4j jars that are duplicated.